nilearn/image/image.py

```diff
@@ -880,6 +880,18 @@ def clean_img(imgs, sessions=None, detrend=True, standardize=True,
     from .image import new_img_like
 
     imgs_ = check_niimg_4d(imgs)
+
+    # Check if t_r is set, otherwise propose t_r from imgs header
+    if low_pass is not None or high_pass is not None:
+        if t_r is None:
+            t_r = imgs.header.get_zooms()[3]
```

Comment:
overwriting t_r which is None to a value of imgs.header.get_zooms()[3] looks dangerous to me.

Reply:
You're right. I will feed it directly to the `format` function.